pkgs/development/ocaml-modules/cudf/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib, fetchzip, stdenv, ocaml, ocamlbuild, findlib, ocaml_extlib, glib, perl, pkg-config, stdlib-shims, ounit }:
+
+stdenv.mkDerivation {
+  pname = "cudf";
+  version = "0.9";
+
+  src = fetchzip {
+    url = "https://gforge.inria.fr/frs/download.php/36602/cudf-0.9.tar.gz";
```

Comment:
For this URL, `fetchurl` is probably fine.

Reply:
Done